src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -150,5 +150,13 @@ public void globalLibrarySuccess() throws Exception {
 
         j.assertLogContains("[nothing here]", b);
         j.assertLogContains("map call(1,2)", b);
+        /* TODO called the wrong way: closure1([a:1])
```

Comment:
And running the same code (i.e., still from raw Pipeline, not the model) but using it from the global library gets me 
```
hudson.remoting.ProxyException: groovy.lang.MissingMethodException: No signature of method: acmeFuncClosure2.call() is applicable for argument types: (java.util.LinkedHashMap, org.jenkinsci.plugins.workflow.cps.CpsClosure2) values: [[a:1, b:2], org.jenkinsci.plugins.workflow.cps.CpsClosure2@ed5c05c]
```
So...yeah. That's busted weird.